samples/swiftui/FirebaseSwiftUIExample/FirebaseSwiftUIExample/App/ContentView.swift

```diff
@@ -0,0 +1,175 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//
+//  ContentView.swift
+//  FirebaseSwiftUIExample
+//
+//  Created by Russell Wheatley on 23/04/2025.
+//
+
+import AppTrackingTransparency
+import FirebaseAuth
+import FirebaseAuthSwiftUI
+import FirebaseFacebookSwiftUI
+import FirebaseGoogleSwiftUI
+import FirebasePhoneAuthSwiftUI
+import FirebaseTwitterSwiftUI
+import FirebaseAppleSwiftUI
+import FirebaseOAuthSwiftUI
+import SwiftUI
+
+struct ContentView: View {
+  init() {
+    let actionCodeSettings = ActionCodeSettings()
+    actionCodeSettings.handleCodeInApp = true
+    actionCodeSettings
+      .url = URL(string: "https://flutterfire-e2e-tests.firebaseapp.com")
+    actionCodeSettings.linkDomain = "flutterfire-e2e-tests.firebaseapp.com"
+    actionCodeSettings.setIOSBundleID(Bundle.main.bundleIdentifier!)
+    let configuration = AuthConfiguration(
+      tosUrl: URL(string: "https://example.com/tos"),
+      privacyPolicyUrl: URL(string: "https://example.com/privacy"),
+      emailLinkSignInActionCodeSettings: actionCodeSettings,
+      mfaEnabled: true
+    )
+    
+    // Create Facebook provider with Limited Login enabled by default
+    let fbProvider = FacebookProviderSwift()
+    fbProvider.setLimitedLogin(true)
+    facebookProvider = fbProvider
```

Comment:
I'm not keen on these changes to authorise user tracking. It should still be self contained within Facebook. 

We should possibly remove the limited login and user tracking all together and check under the hood to see if user tracking is allowed. If so, use classic login, if it isn't allowed, use limited login.

Reply:
fair point

Reply:
I think another thing to note is that _isLimitedLogin_ config in FacebookProviderSwift. if limited login is disabled by the developer and the user denies ATT this would mean facebook sign in is disabled entirely.